### ISSUE TITLE
Doesn't work on Microsoft Edge (44.18362.449.0)

### ISSUE THREAD

**Issue by konclave** [greybax/cyrillic-to-translit-js#92]

**Comment by greybax** [comment]
@konclave could you please share any console errors here?

**Comment by konclave** [comment]
It's hard to get something clear in Edge debugger. It shows error: `SCRIPT1028: SCRIPT1028: Expected identifier, string or number`  on string number 59 of bundle.js:
``` 
_reversedFirstLetters = { ...invert(_firstLetters), "i": "и", "": "" };
```

**Comment by greybax** [comment]
based on this error seems that lodash function `invert` has broken in MS Edge... Will take a look a bit later, but any help is also welcome ;)

**Comment by konclave** [comment]
Seems like I have same kind of troubles to fix this issue as you. I use Mac. =)

**Comment by mezhevikin** [comment]
The demo does not work in Internet Explorer 11.
```
Error: cyrillicToTranslit is undefined
cyrillic-to-translit-js (124,9)
```

**Comment by uqe** [comment]
Fixed #97

**Comment by greybax** [comment]
PR has been failed due to some test errors

**Comment by uqe** [comment]
@greybax sorry, fixed #99
